core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -107,6 +107,9 @@ object ConsumerGroupCommand extends Logging {
         }
       }
       else if (opts.options.has(opts.resetOffsetsOpt)) {
+        if(!opts.options.has(opts.executeOpt))
+          System.out.println("Note: No action will be performed as argument \"" + opts.executeOpt + "\" is missing.")
```

Comment:
Would it make sense to print this on stderr instead? I was thinking that might be the command more unix-friendly (e.g. the offsets can be piped to another command more easily).

Reply:
Indeed, it would be better on stderr, I will update it.